client/src/components/Chat/ChatDetails/ChatDetails.jsx

```diff
@@ -9,7 +9,16 @@ import SendMessage from './SendMessage/SendMessage'
 
 import useStyles from './chatDetailsStyles'
 
-const ChatDetails = ({ authUser, chat, messages, className }) => {
+const ChatDetails = ({
+  authUser,
+  chat,
+  messages,
+  className,
+  messagesLoading,
+  loadContentHandler,
+  ownMessageSend,
```

Comment:
I you were frustrated with "isAllValidationsDone", "ownMessageSend" should also annoy you ;)

Reply:
))